src/main/java/team/easytravel/logic/parser/fixedexpense/EditFixedExpenseCommandParser.java

```diff
@@ -43,30 +44,30 @@ public EditFixedExpenseCommand parse(String args) throws ParseException {
         EditFixedExpenseCommand.EditFixedExpenseDescriptor editFixedExpenseDescriptor =
                 new EditFixedExpenseCommand.EditFixedExpenseDescriptor();
 
-        if (argMultimap.getValue(PREFIX_AMOUNT).isPresent()) {
+        if (argMultimap.getValue(PREFIX_EXPENSE_AMOUNT).isPresent()) {
 
-            if (argMultimap.getValue(PREFIX_CURRENCY).isEmpty()) {
+            if (argMultimap.getValue(PREFIX_EXPENSE_CURRENCY).isEmpty()) {
                 throw new ParseException(EditFixedExpenseCommand.MESSAGE_CURRENCY_NOT_PRESENT);
-            } else if (argMultimap.getValue(PREFIX_CURRENCY).get().toLowerCase().equals("sgd")) {
+            } else if (argMultimap.getValue(PREFIX_EXPENSE_CURRENCY).get().toLowerCase().equals("sgd")) {
```

Comment:
Can just write .ignoreCase() instead. Can follow the code quality

Reply:
Yeah I've changed it alr in my PR. Thanks for pointing it out tho